x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.tsx

```diff
@@ -63,18 +66,23 @@ const RuleActionsOverflowComponent = ({
     () =>
       rule != null
         ? [
-            <EuiContextMenuItem
-              key={i18nActions.DUPLICATE_RULE}
-              icon="copy"
-              disabled={userHasNoPermissions}
-              data-test-subj="rules-details-duplicate-rule"
-              onClick={async () => {
-                setIsPopoverOpen(false);
-                await duplicateRulesAction([rule], [rule.id], noop, dispatchToaster);
-              }}
+            <EuiToolTip
+              position="left"
+              content={getToolTipContent(rule, true, canDuplicateRuleWithActions)}
             >
-              {i18nActions.DUPLICATE_RULE}
-            </EuiContextMenuItem>,
+              <EuiContextMenuItem
+                key={i18nActions.DUPLICATE_RULE}
+                icon="copy"
+                disabled={!canDuplicateRuleWithActions || userHasNoPermissions}
+                data-test-subj="rules-details-duplicate-rule"
+                onClick={async () => {
+                  setIsPopoverOpen(false);
+                  await duplicateRulesAction([rule], [rule.id], noop, dispatchToaster);
+                }}
+              >
+                {i18nActions.DUPLICATE_RULE}
```

Comment:
nit: Could move `EuiToolTip` to wrap this instead of `EuiContextMenuItem` in the event we end up adding a tooltip to the next menu item as well (to prevent the styling issue). This also ensure's all top level components have a `key` as well.